### PR TITLE
Introduce log collection for aks-engine Windows Nodes

### PR DESCRIPTION
**Pull request by adelina-t**

### CODE REVIEW

**Comment by adelina-t** [comment]
related: https://github.com/kubernetes-sigs/windows-testing/pull/127

File: kubetest/azure.go [hunk 1]
```diff
@@ -1056,20 +1064,54 @@ func (c *Cluster) DumpClusterLogs(localPath, gcsPath string) error {
...
+		return nil
 	}
 
-	return control.FinishRunning(exec.Command("bash", "-c", logDumpScript))
+	var errors []string
+	if err := logDumper(); err != nil {
+		errors = append(errors, err.Error())
+	}
+	if err := logDumperWindows(); err != nil {
+		errors = append(errors, err.Error())
```
Comment (feiskyer) [inline kubetest/azure.go]: would this report errors if there're no Windows nodes in the cluster?

Comment (adelina-t) [reply kubetest/azure.go]: No. The collection script will just exit and log that there are no windows nodes it can access.

Comment (feiskyer) [reply kubetest/azure.go]: thanks for confirm

**Pull request merged**